backend/src/email/routes/tests/email-transactional.routes.test.ts

```diff
@@ -866,14 +866,11 @@ describe(`${emailTransactionalRoute}/send`, () => {
   })
 
   const ccValidTests = [
-    ['cc-recipient@agency.gov.sg', 'cc-recipient-2@agency.gov.sg'],
-    JSON.stringify([
-      'cc-recipient@agency.gov.sg',
-      'cc-recipient-2@agency.gov.sg',
-    ]),
+    [['cc-recipient@agency.gov.sg']],
+    [['cc-recipient@agency.gov.sg', 'cc-recipient-2@agency.gov.sg']],
```

Comment:
I'm confused... why did we change the value of the test parameters to 2d arrays?

Reply:
reverted to 3d arrays, apologies for the confusion